iree_tests/pytorch/models/opt-125M/test_cases.json

```diff
@@ -0,0 +1,49 @@
+{
+  "test_cases": [
+    {
+      "name": "splats",
+      "runtime_flagfile": "splat_data_flags.txt",
+      "remote_file_groups": []
+    },
+    {
+      "name": "real_weights",
+      "runtime_flagfile": "real_weights_data_flags.txt",
+      "remote_file_groups": [
+        {
+          "azure_account_url": "https://sharkpublic.blob.core.windows.net",
+          "azure_container_name": "sharkpublic",
+          "azure_base_blob_name": "scotttodd/iree_tests/2024-03-12/opt-125M/",
```

Comment:
@saienduri I just put files here to start/prototype. We can move to a different container with some versioning plan. I haven't put too much thought into that yet.